src/Aquarius.ONE.ClientSDK/Aquarius.ONE.ClientSDK.csproj

```diff
@@ -5,7 +5,7 @@
 	  <!--For Release packages update the VersionPrefix and remove any VersionSuffix-->
 	  <VersionPrefix>13.15.0</VersionPrefix>
 	  <!--For Feature packages update the VersionSuffix with the Jira ticket number-->
-	  <VersionSuffix></VersionSuffix>
+	  <VersionSuffix>prerelease-1</VersionSuffix>

```

Comment:
Not a huge deal but like the comment states, the VersionSuffix value for pre-release versions should contain the Jira ticket number. Something like aqi-3743-01

Reply:
I'd prefer to keep this out of the public NuGet, now that we're using this. I'd vote for changing the comment to not include a Jira ticket number

Reply:
Agreed, when we moved these packages to public nuget we stopped using jira ticket numbers although that perhaps was not communicated.

Reply:
I already did that earlier but as per Jon it is public facing so I did change to prerelease-1. @JonathanRobertson-AQI  correct?